### ISSUE TITLE
Multiline hash attributes shouldn't get flagged for spacing issue

### ISSUE THREAD

**Issue by bfad** [sds/haml-lint#339]
Haml Lint version: 0.37.1
Using the default value for `SpaceInsideHashAttributes`

Running on a file that looks like this:

```haml
%div
  %p{
    "data-stuff": "thing",
    "data-more": "double",
    "another": "fun"
  }
```

I would expect no errors, but I get the following:

```text
test.html.haml:2 [W] SpaceInsideHashAttributes: Hash attribute should start with one space after the opening brace
test.html.haml:2 [W] SpaceInsideHashAttributes: Hash attribute should end with one space before the closing brace
```

**Comment by bfad** [comment]
Looking at the code, I wouldn't mind trying to fix this.
